cmds/boot/systemboot/main.go

```diff
@@ -98,6 +100,36 @@ func runIPMICommands() {
 
 }
 
+func sendBootSEL(sequence string) {
+	var bootErr ipmi.Event
+	var OEM_NTS_TYPE uint8 = ipmi.OEM_NTS_TYPE
+
+	ipmi, err := ipmi.Open(0)
+	if err != nil {
+		log.Printf("Failed to open ipmi device to send SEL %v", err)
+		return
+	}
+	defer ipmi.Close()
+
+	switch sequence {
+	case "fbnetboot":
+		bootErr.RecordID = 0x0000
+		bootErr.RecordType = OEM_NTS_TYPE
```

Comment:
Can this be `bootErr.RecordType = ipmi.OEM_NTS_TYPE`?

Reply:
Modified to be ipmi const.